src/main/java/Deadline.java

```diff
@@ -0,0 +1,27 @@
+public class Deadline extends Todo {
+    String by;
+
+    public String getBy() {
+        return by;
+    }
+
+    public void setBy(String by) {
+        this.by = by;
+    }
+
+    Deadline(String description, String by) {
```

Comment:
```suggestion
public Deadline(String description, String by) {
```
Perhaps you can amend the method by adding a public for accessibility?